src/pages/currency-converter.tsx

```diff
@@ -0,0 +1,71 @@
+import CurrencyService, { CurrencyMatrix } from 'services/currency';
+import Layout from 'components/Layout'
+import styles from 'styles/Home.module.scss'
+import * as E from 'fp-ts/lib/Either';
+import React from 'react';
+
+type Props = {
+	topCurrencies: CurrencyMatrix | null,
+	otherCurrencies: CurrencyMatrix | null,
+	error: string | null,
+}
+
+export async function getStaticProps() {
+	const currencyRes = await CurrencyService.fetchCurrenciesListByRelevance();
+
+	console.log('STATIC_PROPS_DATA', currencyRes);
+
+	return {
+	  props: {
+			error: E.isLeft(currencyRes) ? currencyRes?.left.message : null,
+			otherCurrencies: E.isRight(currencyRes) ? currencyRes?.right?.left : null,
+			topCurrencies: E.isRight(currencyRes) ? currencyRes?.right?.right : null,
+	  },
+	}
+}
+
+export default function CurrencyConverter({ topCurrencies, otherCurrencies, error }: Props) {
```

Comment:
we usually use this notation: 
```
import { FC } from 'react';
export const CurrencyConverter: FC<CurrencyConverterProps>({ topCurrencies, otherCurrencies, error }) => {...}
```
and we don't use default export.

Reply:
@yugo-eliatrop I converted all components to use FC/export, but in the page files Next throws an error `The default export is not a React Component` if I don't use `export default`. :/

Reply:
Partially fixed in [caed15a](https://github.com/rafaelpradella/pradella-fp-page/pull/2/commits/caed15a45cfd8eb5581b6dec633557ff9797822b)